### PR TITLE
Tdl 24463/setup circleci config

### PR DESCRIPTION
**Pull request by rdeshmukh15**
# Description of change
- setup circleci project for tap-frontapp

# Manual QA steps
 - 
 
# Risks
 - low
 
# Rollback steps
 - revert this branch

### CODE REVIEW

File: setup.py [hunk 1]
```diff
@@ -4,7 +4,7 @@
 
 setup(
     name="tap-frontapp",
-    version="1.1.0",
+    version="1.1.1",
```
Comment (Vi6hal) [inline setup.py:7]:
```suggestion
    version="1.1.0",
```

Comment (Vi6hal) [reply setup.py:7]: version change not necessary for circle-ci config

File: tap_frontapp/__init__.py [hunk 2]
```diff
@@ -1,5 +1,5 @@
 #!/usr/bin/env python3
-
+# pylint: disable=E0602
```
Comment (Vi6hal) [inline tap_frontapp/__init__.py:2]: can you address the issue that caused this pylint error?
its not a good practice to disable Error/fatal pylint messages.

**Review by Vi6hal — CHANGES_REQUESTED** [review]

File: setup.py (hunk 1 above)
Comment (rdeshmukh15) [reply setup.py:7]: releasing it along with #18 hence, added

**Review by Vi6hal — APPROVED** [review]

**Pull request merged**